cstore_fdw.c

```diff
@@ -254,6 +248,139 @@ cstore_ddl_event_end_trigger(PG_FUNCTION_ARGS)
 
 
 /*
+ * RegisterCStoreTable registers provided cstore table relation id by
+ * adding a new row to pg_cstore_tables.
+ */
+static void
+RegisterCStoreTable(Oid relationId, Oid fileNodeOid)
```

Comment:
Alternatively, we could use `SPI_Connect()`, `SPI_Execute()`, ... to implement this function and the unregister function if that makes things simpler.